src/main/java/com/trailfinder/dto/EventDTO.java

```diff
@@ -17,8 +17,8 @@
 	private double distance;
 	private LocalDateTime eventStart;
 	private LocalDateTime eventEnd;
-	private double Latitude;
-	private double Longitude;
+	private double latitude;
```

Comment:
This is inline with Java naming conventions: variables should be lowerCamelCase.